xml/obs_best_practice_image_templates.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+
+<section id="image_templates_how_to">
+ <title>Image templates</title>
+ <xi:include href="obs_image_templates.xml" xpointer="image_template_introduction_paragraph" xmlns:xi="http://www.w3.org/2001/XInclude"/>
+ <para>
+  How you can create your own image templates will be shown here.
+ </para>
+ <figure>
+  <title>OBS templates page</title>
+  <mediaobject>
+   <imageobject>
+    <imagedata fileref="image_templates_template_page_placeholder.png" />
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <section>
+  <title>Creating your own image templates</title>
+  <para>
+   Create a subproject of your home project.
+  </para>
+  <figure>
+   <title>Form for creating image template subproject</title>
+   <mediaobject>
+    <imageobject>
+     <imagedata fileref="image_templates_project.png" />
+    </imageobject>
+   </mediaobject>
+  </figure>
+  <note>
+   <para>
+    Published image templates are fetched via a project's attribute. Any package
+    container living in a published project will be visible on the image
+    templates page.
+   </para>
+  </note>
+  <para>
+   Within that project create a new package. That will be your actual image template.
+  </para>
+  <figure>
+   <title>New image template</title>
+   <mediaobject>
+    <imageobject>
+     <imagedata fileref="image_templates_package.png" />
+    </imageobject>
+   </mediaobject>
+  </figure>
+  <para>
+   Add sources for your image configuration.
+  </para>
+  <figure>
+   <title>Overview of sources of custom image template</title>
```

Comment:
* The two `of` here are... not sure this looks somehow strange. Can you maybe think a bit about a different naming?
* Please also use the capitalization style then: http://docserv.suse.de/documents/Styleguide/docu_styleguide/single-html/#sec.capitalization